### ISSUE TITLE
Fresh Install-- reading GetTime errors

### ISSUE THREAD

**Issue by davthomaspilot** [rstrouse/nodejs-poolController-dashPanel#84]
my SD card got corrupted, so I have to start over.

Fresh Bullseye install, following the instructions here:

https://github.com/rstrouse/nodejs-poolController-dashPanel/wiki/nodejs-poolController-dashPanel-setup-and-configuration-on-a-Raspberry-Pi-for-the-novice

Getting these messages:

Error triggering nixie schedules: Cannot read properties of null (reading 'getTime')

I have REM on another pi, so it's still running.  But, I don't recall how to have REM find the njsPC server.  

![image](https://github.com/rstrouse/nodejs-poolController-dashPanel/assets/6830886/25dba03e-1a67-40dc-ba0f-c026ef436f7a)

The + sign next to the Connections seems intuitive, but I see two things there already.  (The ip and mac of the pi running njsPc has not changed)  I can't find it in the documentation--a hint would be appreciated.

Thanks!

**Comment by davthomaspilot** [comment]
Figured out how to reconfigure the chemistry stuff in njsPC.

But, still don't know whether I should worry about the getTime errors or how to fix.

**Comment by davthomaspilot** [comment]
Well, I thought I had the configuration for REM figured out, but something must be wrong.

I'm getting pump and probe faults indicated on the dashPanel, but the  ph and ORP readings look reasonable.  I manually added chlorine and acid, and the readings changed as expected (haven't tried using the dosing pumps yet)

I'm probably not filling this out correctly:

![image](https://github.com/rstrouse/nodejs-poolController-dashPanel/assets/6830886/fe5a8eac-ca79-4ccf-bf33-de07e163f4d3)

I check REM feed under both PH and ORP but when I revisit the sheet it is unchecked.

The REM pi must be communicating with nodeJS pi, since the ORP and ph readings are reasonable and changing.

Do I need something in the Connection and Device fields rather than just clicking REM?

**Comment by tagyoureit** [comment]
Yes, you need to tell njsPC which connection and device for of the probes, pumps, etc are the correct ones.

**Comment by davthomaspilot** [comment]
So, how is that done?  An IP address for the REM pi?

**Comment by davthomaspilot** [comment]
It's set up in the REM pi correctly, I think.

Nothing has changed on the REM pi.  Flow sensor and pump GPIO are still correctly defined, as is the I2C data for the Atlas PH and ORP sensors.

So, what do I need to do on the njsPC side for those entries?

**Comment by davthomaspilot** [comment]
Ok, figured it out.

I needed to find the REM controller from the njsPC first.  Then, the drop downs on the hardware page have entries I can choose.

What about these messages?

Error triggering nixie schedules: Cannot read properties of null (reading 'getTime')

**Comment by tagyoureit** [comment]
The dropdowns should have the devices that are set up in REM.  Check the [wiki](https://github.com/tagyoureit/nodejs-poolController/wiki/DIY-REM-Chem-Controller#configuring-rem-software).  The errors will likely go away when the software is configured properly.
